bootloader/src/pyi_pythonlib.c

```diff
@@ -454,7 +454,7 @@ pyi_pylib_start_python(ARCHIVE_STATUS *status)
      * That is, the APIs use the string pointer as given and will neither copy
      * its contents nor free its memory.
      *
-     * NOTE: Statics are zero-initialized. */
+     * NOTE: Statistics are zero-initialized. */
```

Comment:
The original is correct here (although it would be better written as `Static variables are zero-initialized`).